src/main/java/com/wavefront/WavefrontTraceLoader.java

```
@@ -63,7 +63,22 @@ void setupSenders() throws IOException {
       wfSender = wfClientFactory.getClient();
       WavefrontSpanReporter wfSpanReporter = new WavefrontSpanReporter.Builder().build(wfSender);
       wfSpanReporter.setMetricsReporter(new WavefrontInternalReporter.Builder().build(wfSender));
-      spanSender = new SpanSender(wfSender, generatorConfig.getSpansRate(), dataQueue);
+
+      WavefrontSender statSender;
+      WavefrontClientFactory statClientFactory = new WavefrontClientFactory();
+      if(applicationConfig.getStatServer() != null && applicationConfig.getStatToken() != null) {
+        statClientFactory.addClient("https://" + applicationConfig.getStatToken() +
+            "@" + applicationConfig.getStatServer());
+      }
+      else {
+        statClientFactory.addClient("https://" + applicationConfig.getToken() +
```

What if a user uses `proxy` for ingestion and didn't provide a `serverName` and `token`?
I guess that `statSender` should be `null` in this case.

---

So if neither server nor statServer have been provided, statistics will not be sent anywhere?

---

Where to send it? 
If `reportStat=true` you can throw an exception and terminate the App. or just output it in console, like it was done before.